modules/pom.xml

```diff
@@ -35,6 +35,7 @@
         <module>schedulers</module>
         <module>scripting</module>
         <module>scripting-jruby</module>
+        <module>scripting-plugin</module>
```

Comment:
We already have file-extension-common inside the modules group. I can move it, but moving it to extensions is not right to me, as this is not an extension